types/meteor/globals/reactive-dict.d.ts

```diff
@@ -0,0 +1,15 @@
+declare var ReactiveDict: ReactiveDictStatic;
+declare interface ReactiveDictStatic {
+    new (name?: string, initialValue?: EJSONable): ReactiveDict;
+}
+declare interface ReactiveDict {
+    setDefault(key: string, value: EJSONableProperty): void;
+    setDefault(object: EJSONableProperty): void;
+    set(key: string, value: EJSONableProperty): void;
+    set(object: EJSONableProperty): void;
+    get(key: string): EJSONableProperty;
+    equals(key: string, value: string | number | boolean | undefined): boolean;
+    all(): EJSONable;
+    clear(): void;
+    destroy(): boolean;
```

Comment:
It should have a `void` return type. [Source](https://github.com/meteor/meteor/blob/71f67d9dbac34aba34ceef733b2b51c2bd44b665/packages/reactive-dict/reactive-dict.js#L285-L298). There is also `delete(key: string): boolean` ([source](https://github.com/meteor/meteor/blob/71f67d9dbac34aba34ceef733b2b51c2bd44b665/packages/reactive-dict/reactive-dict.js#L260-L283)), but it's not in [Meteor Docs](https://docs.meteor.com/api/reactive-dict.html), therefore I'd leave it for now.